pages/docs/community/tooling.md

```diff
@@ -111,3 +111,16 @@ The following is a list of tools that validate AsyncAPI documents.
 | [AsyncAPI Parser](https://github.com/asyncapi/parser) | It parses and validates AsyncAPI documents. | Go
 | [Spectral](https://github.com/stoplightio/spectral) | A command-line linter for AsyncAPI & OpenAPI documents. | Javascript
 | [AMF](https://github.com/aml-org/amf) | Unified RAML / OAS / AsyncAPI parser and validator, including linting | ScalaJS / JVM and JS support
+
+---
+
+<DocsButton
+  suggestions={[
+    {
+      href: '/docs//docs/specifications/v2.0.0',
+      title: 'Specifications - Release 2.0.0',
+      type:'back',
+      classname: 'mr-4',
```

Comment:
I think that we should avoid using `className` because people need to define that gap between back and next button manually and it's redundant in all pages. We should handle that in the `DocsButton` component to improve UX of that component.